docker-build-system/package.sh

```diff
@@ -81,10 +83,34 @@ elif [ "${DEBRPM}" == "rpm" ]; then
 		--rpm-os linux \
 		-p ${BUILDDIR}/target/release/zombodb_${OSNAME}_${PGVER}-${VERSION}_1.x86_64.rpm \
 		-a x86_64 \
-		. || exit 1	
+		. || exit 1
+
+elif [ "${PKG_FORMAT}" == "apk" ]; then
+	fpm \
+		-s dir \
+		-t apk \
+		-n zombodb-${PGVER} \
+		-v ${VERSION} \
+		--depends postgresql \
+		-p ${BUILDDIR}/target/release/zombodb_${OSNAME}_${PGVER}-${VERSION}.$(uname -m).apk \
+		-a $(uname -m) \
+		./home/mathieu/postgresql/${PGVER}/build/share/extension=/usr/share/postgresql/ \
```

Comment:
these paths shouldn't be hardcoded.

Reply:
oops, that’s fixed

Reply:
The `fpm` calls for .deb and .rpm just use `.` (the script changes into the package directory ahead of all this).  Can we just do that here too?

The intent is that `fpm` will then just pickup whatever `cargo pgx package` might drop in there now and in the future.

If that'll work here (don't see why it won't), then I'll be happy to merge!

Reply:
I can’t do that here for 2 reasons:

* because does not support installing multiple version of postgres at the same time it does not have the postgres version in the path, but I can’t install each version of postgres from source in the same folder
* the "standard" postgres package (installed with `apk add postgres`) and the official postgres alpine image https://hub.docker.com/_/postgres don’t install postgres at the same place so even if I did not have the first reason I’ll have to rewrite the path for at least one of the build

but I agree that listing multiple folders however is a pain, I might be able to do something about that. I’d like to simplify that. I’ll give it a try later

Reply:
wait a minute.  If there's official Postgres images, shouldn't we be using those?  Looks like there's official images for 10,11,12,and 13.

we'd have to rejigger some things with build.sh, but isn't that a saner approach than a generic Alpine image?

Reply:
that’s another way yes. I thought it maybe would miss some dev files from postgres but it’s working. I’ll make another PR to compare the two approaches. as you said the challenge will be to update build.sh to build for each postgres version in a different image instead of doing everything in the same one. I’ll start with just one version and see how it goes after

Reply:
Yeah. Cool. I appreciate your tenacity!

Reply:
to be clear here, an "official" image here does not necessarily means official from Postgres or from Alpine, it means official from Docker point of view, more info here : https://docs.docker.com/docker-hub/official_images/

Reply:
Yeah. True.  Are there usage stats we can look at to decide if it’s even worth perusing?

Reply:
I found this, using docker hub's API:

```json
{
  "user": "library",
  "name": "postgres",
  "namespace": "library",
  "repository_type": "image",
  "status": 1,
  "description": "The PostgreSQL object-relational database system provides reliability and data integrity.",
  "is_private": false,
  "is_automated": false,
  "can_edit": false,
  "star_count": 8586,
  "pull_count": 2147483647,
  "last_updated": "2020-11-18T09:02:51.366319Z",
  "is_migrated": false,
  "collaborator_count": 0,
  "affiliation": null,
  "has_starred": false,
  "full_description": "<snip>>",
  "permissions": {
    "read": true,
    "write": false,
    "admin": false
  }
}
```

However, that pull_count value looks suspicious.  Dunno if bugged or if the Postgres image has just maxed out the counter.  The "ubuntu" image says the same, so I dunno.

Reply:
yeah I think it gonna be hard to find interesting stats, the postgres image on docker come in two flavors, debian and alpine, so the total count wouldn’t tell us which one is used. 
and for the alpine base image, the same apply, there’s a lot of pull on this image, but it’s used for much more than postgres

my guts tells me that using the base image for postgres is not widespread at all. it’s much more easier to use the postgres image directly. so I don’t think we should take too much time trying to support that. if it become a wanted feature it could be added at that time